omnibus/python-scripts/postinst.py

```diff
@@ -0,0 +1,130 @@
+"""
+This module provides functions for managing Datadog integrations and Python dependencies.
+
+Functions:
+- run_command(command): Executes a shell command and returns its output.
+- create_integrations_file(directory): Creates a file listing currently installed Datadog integrations.
+- create_dependencies_file(directory): Creates a file listing currently installed Python dependencies, excluding Datadog packages.
+- install_integrations(directory): Installs Datadog integrations listed in a file.
+- install_dependencies(directory): Installs Python dependencies listed in a file.
+
+Usage:
+- The script should be run with a single argument specifying the installation directory.
+- Example: `python script.py /path/to/install/dir`
+"""
+
+import os
+import subprocess
+import shutil
+import sys
+
+def run_command(command):
+    """
+    Execute a shell command and return its output.
+
+    Args:
+        command (str): The shell command to execute.
+
+    Returns:
+        str: The standard output of the command.
+
+    Raises:
+        subprocess.CalledProcessError: If the command exits with a non-zero status.
+    """
+    result = subprocess.run(command, shell=True, text=True, capture_output=True, check=True)
+    return result.stdout.strip()
+
+def create_integrations_file(directory):
+    """
+    Create a file listing the currently installed Datadog integrations.
+
+    This function runs the `datadog-agent integration freeze` command, sorts the output,
+    and writes it to a file named '.datadog_requirements.txt' in the specified directory.
+    It also changes the file ownership to 'dd-agent:dd-agent'.
+
+    Args:
+        directory (str): The directory where the integrations file will be created.
+    """
+    print("Creating integrations file")
+    datadog_req_file = os.path.join(directory, '.datadog_requirements.txt')
+    
+    with open(datadog_req_file, 'w', encoding='utf-8') as f:
+        output = run_command('datadog-agent integration freeze | sort')
```

Comment:
We sort the output to make it easier to generate the diff right?
I think it would be better if we could sort in python as well and not rely on the `sort` utility in case it's not available in the system.

Reply:
Makes sense. I'll rewrite it.